cpp/src/gandiva/function_registry_string.cc

```diff
@@ -65,6 +65,9 @@ std::vector<NativeFunction> GetStringFunctionRegistry() {
       UNARY_SAFE_NULL_NEVER_BOOL_FN(isnull, {}),
       UNARY_SAFE_NULL_NEVER_BOOL_FN(isnotnull, {}),
 
+      NativeFunction("chr", {}, DataTypeVector{int32()}, utf8(), kResultNullIfNull,
```

Comment:
Hive implements this for bigint and double. I dont see the Hive documentation implementing this for int

Reply:
Changed this to bigint and double